include/swift/AST/ModuleDependencies.h

```diff
@@ -805,7 +805,7 @@ class ModuleDependencyInfo {
     return {};
   }
 
-  std::vector<std::string> getCommandline() const {
+  const ArrayRef<std::string> getCommandline() const {
```

Comment:
`const` `ArrayRef` doesn't really make sense. Just use `ArrayRef`. Same for other places.